compiler/rustc_smir/src/rustc_smir/convert/ty.rs

```diff
@@ -690,6 +689,9 @@ impl<'tcx> Stable<'tcx> for ty::ClauseKind<'tcx> {
             ClauseKind::ConstEvaluatable(const_) => {
                 stable_mir::ty::ClauseKind::ConstEvaluatable(const_.stable(tables))
             }
+            ClauseKind::HostEffect(..) => {
+                todo!()
```

Comment:
cc @rust-lang/project-stable-mir

Reply:
I'll leave this unimplemented currently, at least until we get consensus that this is something we want :D